db/migrate/20210916124318_devise_create_users.rb

```diff
@@ -30,10 +30,10 @@ def change
       # t.inet     :last_sign_in_ip
 
       ## Confirmable
-      t.string   :confirmation_token
-      t.datetime :confirmed_at
-      t.datetime :confirmation_sent_at
-      t.string   :unconfirmed_email # Only if using reconfirmable
+      # t.string   :confirmation_token
+      # t.datetime :confirmed_at
+      # t.datetime :confirmation_sent_at
+      # t.string   :unconfirmed_email # Only if using reconfirmable
```

Comment:
I think this migration has been applied to your production database. You should create a new migration file for removing these attributes.